scripts/devaccess/test_devget.py

```diff
@@ -145,5 +145,16 @@ def test_invalid_username(self):
         with self.assertRaises(DevGetError):
             dg = DevGet()
 
+    @mock.patch('devaccess_wrap.WhitelistedCommands.rpm_qa')
+    def test_user_with_no_defined_roles(self, mock_rpm_qa):
+        ''' Test user with no defined roles (ie only default role membership) '''
+        os.environ['SSH_ORIGINAL_COMMAND'] = 'rpm -qa'
+        sys.argv = ['devaccess_wrap.py', 'READ_SSH', 'noroleuser']
+        dg = DevGet()
+
+        mock_rpm_qa.side_effect = ['kernel-3.12.22']
```

Comment:
right, it's just some dummy output.
the test is to make sure that the function gets called because 'rpm -qa' is in the default list of allowed commands.